.npmignore

```diff
@@ -0,0 +1,2 @@
+# We doesn't wan to publish the logo and other
+assets
```

Comment:
You can add the logos inside the `.github` to avoid being included in the package.

Reply:
Yeah that true. But I chose to create a separate dir to allow user to easily found that.